### DISCUSSION TITLE
JavaScript Clipboard Stuff

### DISCUSSION THREAD

**Discussion by giscus** [cfjedimaster/raymondcamden2023#40] [Comments]
# JavaScript Clipboard Stuff

Just a pointer to my series on JavaScript clipboard reading and writing.

https://www.raymondcamden.com/2024/10/14/javascript-clipboard-stuff

**Comment by bobmonsour** [comment]
Just wanted to share that I used the same image on the first blog post on my site: https://www.bobmonsour.com/posts/is-this-thing-on/

**Reply by cfjedimaster** [reply]
Nice! I usually credit my Unsplash images the first time I use them - this is one I've used in a previous post so I didn't do it here.

**Reply by cfjedimaster** [reply]
And now I feel a bit guilty, just pushed an update with a link for his credit. :)
